src/Rules/AlphaNew

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rakit\Validation\Rules;
+
+use Rakit\Validation\Rule;
+
+class AlphaNew extends Rule
+{
+
+    protected $message = "The Place Of Birth only allows alphabet and numeric";
+
+    public function check($value)
+    {
+       
+
+        return preg_match('/^[\pL\pM 0-9 \,\.\-\/]+$/u', $value) > 0;
+        
+    }
+
+}
+/*
+Rule to accept alphabet or  numeric or both with or without  (, . / - ' ') 
```

Comment:
By what standards this comment is on the bottom of file?

That is not place for comment like this, code should be self-documenting. If it's not write comment above class but only with technical details.